Checkbox fix- validate for "go" button on Results1.php

emanguy opened the pull request:
This fixes issue #15 . Highlights of the changes in this branch are:
- Change `printAdvisors($currChoice)` in Utilities.php to have advisor boxes checked by default
- Add an ID of `btnGo` to the submit button in Results1.php
- Submit button in Results1.php now disabled by default
- Add a `requiredAtLeastOne` attribute to all individual advising and group advising checkboxes in Results1.php
- Add a script to the end of Results1.php to change the disabled state of the submit button when individual advising or group advising checkboxes are checked

P.S. It's really hard to test my code when the directory structure is wrong and I don't have your CommonMethods.php so I can access the database @jadedtdt

emanguy committed:
Make single-parametrized printAdvisors have checked teacher boxes by default

emanguy committed:
Add JS validator code to Results1, add attributes to inputs

jadedtdt commented:
I created a branch fixing the CommonMethods, not sure how to merge our two branches together though.

jadedtdt commented:
I guess merge mine and I'll merge yours?

emanguy committed:
Tidy up comments in Results1, add break to JS for loop

emanguy committed:
Remove checkboxes in Results2 and clean up display

This commit removes the checkboxes in Results2.php and prevents the "individual advising"
or "group advising" labels from being printed if the respective box wasn't checked on the previous page.
w

emanguy commented:
My latest commit includes the proposed changes in #13 .
